app/controllers/resend_confirmation_email_controller.rb

```diff
@@ -41,7 +41,7 @@ def send_emails
 
   def emails_to_contact
     emails = [registration.applicant_email, registration.contact_email]
-    emails.delete(WasteExemptionsEngine.configuration.assisted_digital_email)
+    emails.compact!
     emails.uniq
   end
```

Comment:
looks like you can lose the assignment here, so you could just write (note the removal of the `!` in `compact!`):

```ruby
def emails_to_contact
  [registration.applicant_email, registration.contact_email]
    .delete(WasteExemptionsEngine.configuration.assisted_digital_email)
    .compact
    .uniq
end
```

Reply:
That looks good thank you